env.js

```diff
@@ -1,13 +1,14 @@
 var log = require('loglevel');
 
 module.exports = function() {
-  if (process.env.COUCH2PG_DEBUG) {
+  if (process.env.COUCH2PG_DEBUG || true) {
```

Comment:
Nope, intentional. The debug output is useful, and when this is properly deployed it should go into a log file.

At some point if we cbf we can replace this env var with an actual minimum level value that we default to debug or something.